src/sql/src/statements.rs

```diff
@@ -145,6 +145,10 @@ macro_rules! parse_number_to_value {
                 let n  = parse_sql_number::<i64>($n)?;
                 Ok(Value::Timestamp(Timestamp::new(n, t.unit())))
             },
+            ConcreteDataType::Decimal128(_) => {
+                // TODO(QuenKar): parse decimal128 string with precision and scale
+                unimplemented!("insert Decimal128 is not supported yet")
+            }
```

Comment:
Don't panic. You can leave a todo here.
```suggestion
            // TODO(QuenKar): parse decimal128 string with precision and scale
```